test/test.bats

```diff
@@ -76,7 +76,9 @@ load test_helper
 
 @test "ldapsearch database from created volumes" {
 
-  rm -rf VOLUMES && mkdir -p VOLUMES/config VOLUMES/database
+  rm -rf VOLUMES || sudo rm -rf VOLUMES
```

Comment:
This "sudo" change is not really required, but could help to recover from a previous failed/aborted "make test".